src/main/resources/js/condec.knowledge.page.js

```diff
@@ -40,8 +40,8 @@
 					isSelected = "selected ";
 				}
 			}
-			jQuery("select[name='knowledge-type-dropdown-overview']")[0].insertAdjacentHTML("beforeend", "<option "
-			        + isSelected + " value='" + knowledgeTypes[index] + "'>" + knowledgeTypes[index] + "</option>");
+			jQuery("select[name='nowledge-type-dropdown-overview']")[0].insertAdjacentHTML("beforeend", "<option "
```

Comment:
@cocoon-it , the "k" is missing here. That's why the overview tab is not working correctly. Can you please fix this? Thanks!